src/translations/en.json

```diff
@@ -2033,50 +2040,60 @@
+              "content": "Content",
+              "description": "The Markdown card is used to render Markdown."
             },
             "media-control": {
-              "name": "Media Control"
+              "name": "Media Control",
+              "description": "The Media Control card is used to display Media Player entities on an interface with easy to use controls."
             },
             "picture": {
-              "name": "Picture"
+              "name": "Picture",
+              "description": "The Picture card allows you to set an image to use for navigation to various paths in your interface or to call a service."
             },
             "picture-elements": {
-              "name": "Picture Elements"
+              "name": "Picture Elements",
+              "description": "The Picture Elements card is one of the most versatile types of cards. The cards allow you to position icons or text and even services! On an image based on coordinates."
             },
             "picture-entity": {
-              "name": "Picture Entity"
+              "name": "Picture Entity",
+              "description": "The Picture Entity card displays an entity in the form of an image. Instead of images from URL, it can also show the picture of Camera entities."
             },
             "picture-glance": {
-              "name": "Picture Glance"
+              "name": "Picture Glance",
+              "description": "The Picture Glance card shows an image and corresponding entity states as an icon. The entities on the right side allow toggle actions, others show the more information dialog."
             },
             "plant-status": {
               "name": "Plant Status",
-              "description": "A card for all the lovely botanists out there."
+              "description": "The Plant Status card is for all the lovely botanists out there."
             },
             "sensor": {
               "name": "Sensor",
               "graph_detail": "Graph Detail",
-              "graph_type": "Graph Type"
+              "graph_type": "Graph Type",
+              "description": "The Sensor card gives you a quick overview of your sensors state with an optional graph to visualize change over time."
             },
             "shopping-list": {
               "name": "Shopping List",
-              "description": "The Shopping List Card allows you to add, edit, check-off, and clear items from your shopping list.",
+              "description": "The Shopping List card allows you to add, edit, check-off, and clear items from your shopping list.",
               "integration_not_loaded": "This card requires the `shopping_list` integration to be set up."
             },
             "thermostat": {
-              "name": "Thermostat"
+              "name": "Thermostat",
+              "description": "The Thermostat card gives control of your Climate entity. Allowing you to change the temperature and mode of the entity"
```

Comment:
```suggestion
              "description": "The Thermostat card gives control of your climate entity. Allowing you to change the temperature and mode of the entity."
```

Reply:
You don't think the name of the entity should be capitalized?

Reply:
Since its a type of entity Climate, Media Player, etc I capitalized them on purpose

Reply:
I dunno.. it is not referring to a specific entity or brand, that was kinda my way of thinking in this case.

Reply:
Since we don't have guidelines on this, both would be OK. So, whatever fits you. I just came across it while adding full stops at the end of the sentences.

Reply:
Gotcha I will update them to be lower case. I will go with your method as you are the documentation master. Especially since I want to update the Lovelace Docs to these descriptions when we finish this PR